go/vt/vttablet/tabletmanager/vreplication/framework_test.go

```diff
@@ -72,10 +72,10 @@ type LogExpectation struct {
 var heartbeatRe *regexp.Regexp
 
 // setFlag() sets a flag for a test in a non-racy way:
-//	* it registers the flag using a different flagset scope
-//	* clears other flags by passing a dummy os.Args() while parsing this flagset
-//	* sets the specific flag, if it has not already been defined
-//	* resets the os.Args() so that the remaining flagsets can be parsed correctly
+//   - it registers the flag using a different flagset scope
+//   - clears other flags by passing a dummy os.Args() while parsing this flagset
+//   - sets the specific flag, if it has not already been defined
+//   - resets the os.Args() so that the remaining flagsets can be parsed correctly
```

Comment:
nit: looks like unrelated change.

Reply:
Was due to pre-commit hook forcing me to accept these formatting changes. Should I run with `git commit --no-verify`, or maybe my local tooling is out of date?

Reply:
to my understanding you did not change those packages so linter should not have failed for them.

Reply:
🤷 maybe they did at one point, which triggered the hook to complain, and then they ended throwing away the change that caused this but keeping the formatting change

Reply:
I think what happened is:

 1. I merged `main` into my branch
 2. There was a merge conflict, which left a bunch of upstream changes in the git staging area
 3. I resolved those conflicts and tried to commit
 4. Linting reformatted a bunch of the files from step 2.